### ISSUE TITLE
junos path show gives exception

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#159]
**Describe the bug**
```
(suzieq-ITL68y-d-py3.7) jpiet@a2:~/suzieq$ python3 suzieq/cli/suzieq-cli -c ~/configs/junos.yml
path show --dest=10.10.1.10 --src=10.10.2.20  --namespace=junos
                                                error
0  ERROR: 'PathObj' object has no attribute '_arp_df'
```

**Comment by jopietsch** [comment]
see the same problem with nxos

**Comment by ddutt** [comment]
Fixed with 81efad952cf28aaf0ef64cd219264959d3e6afa9
